core/helpers/EEH_File.helper.php

```diff
@@ -136,20 +138,20 @@ private static function loadWpFileSystem()
              * and there may be troubles if the WP files are owned by a different user
              * than the server user. But both of these issues should exist in 4.4 and earlier too
              */
-            if (false && ! did_action('wp_loaded')) {
-                $msg =
-                    __(
-                        'An attempt to access and/or write to a file on the server could not be completed due to a lack of sufficient credentials.',
-                        'event_espresso'
-                    );
-                if (WP_DEBUG) {
-                    $msg .= '<br />' . __(
-                        'The WP Filesystem can not be accessed until after the "wp_loaded" hook has run, so it\'s best not to attempt access until the "admin_init" hookpoint.',
-                        'event_espresso'
-                    );
-                }
-                EE_Error::add_error($msg, __FILE__, __FUNCTION__, __LINE__);
-            }
+            // if (false && ! did_action('wp_loaded')) {
+            //     $msg =
+            //         esc_html__(
+            //             'An attempt to access and/or write to a file on the server could not be completed due to a lack of sufficient credentials.',
+            //             'event_espresso'
+            //         );
+            //     if (WP_DEBUG) {
+            //         $msg .= '<br />' . esc_html__(
+            //             'The WP Filesystem can not be accessed until after the "wp_loaded" hook has run, so it\'s best not to attempt access until the "admin_init" hookpoint.',
+            //             'event_espresso'
+            //         );
+            //     }
+            //     EE_Error::add_error($msg, __FILE__, __FUNCTION__, __LINE__);
+            // }
```

Comment:
I had added that check when I first created the class because you're not supposed to try and use the WP filesystem prior to 'wp_loaded', but Michael commented this out because he needed to do so earlier during activations (iirc). In order to fix that, we would need to run the activation logic at a later point in the request (and possibly on a separate request), which is doable and likely a good change, but will require a fair amount of work so I have just kept this commented out as a reminder.